mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -219,6 +220,14 @@ namespace mbf_abstract_nav
      */
     void setVelocityCmd(const geometry_msgs::TwistStamped &vel_cmd_stamped);
 
+    /**
+     * @brief Check if the robot's actual velocity is ignoring the cmd_vel
+     * @param robot_stopped true if robot is stopped, false otherwise
+     * @param cmd_velocity the commanded velocity being published to the robot
+     * @return true if there the ignoring time exceeded tolerance, false otherwise
```

Comment:
```suggestion
     * @return true if velocity is being ignored longer than tolerance
```
F: Some of the comments' grammar throughout the code are rather questionable -- please re-read and rephrase them. (feel free to also rephrase my suggestion above)